commit 39a32d36fc465c4e70f13192eea380e518ba6e8a
Author: Jia Tan <jiat0218@gmail.com>
Date:   Fri Jul 21 18:05:44 2023 +0800

    Tests: Skip .lz files in test_files.sh if not configured.
    
    Previously if the lzip decoder was not configured then test_files.sh
    would pass the lzip tests instead of skipping them.

diff --git a/tests/test_files.sh b/tests/test_files.sh
index e42af918..60402e73 100755
--- a/tests/test_files.sh
+++ b/tests/test_files.sh
@@ -174,7 +174,7 @@ done
 # .lz #
 #######
 
-if grep 'define HAVE_LZIP_DECODER' ../config.h > /dev/null ; then
+if have_feature LZIP_DECODER ".lz files" ; then
 	for I in "$srcdir"/files/good-*.lz
 	do
 		if test -z "$XZ" || "$XZ" -dc "$I" > /dev/null; then